src/StrTools.php

```diff
@@ -117,46 +133,46 @@ public static function markdownToMessageEntities(string $markdown): \danog\Madel
     /**
      * Convert a message and a set of entities to HTML.
      *
-     * @param list<array{_: string, offset: int, length: int, ...}> $entities
+     * @param list<MessageEntity> $entities
      * @param bool $allowTelegramTags Whether to allow telegram-specific tags like tg-spoiler, tg-emoji, mention links and so on...
      */
     public static function entitiesToHtml(string $message, array $entities, bool $allowTelegramTags = false): string
     {
         $insertions = [];
```

Comment:
Could you please convert the values $entities to objects if they're arrays?

Reply:
its array of MessageEntity objects